malcolm/controllers/scanpointtickercontroller.py

```diff
@@ -0,0 +1,58 @@
+import numpy
+import time
+
+from malcolm.core.attribute import Attribute
+from malcolm.core.controller import Controller
+from malcolm.core.mapmeta import REQUIRED
+from malcolm.core.method import takes, Method
+from malcolm.core.pointgeneratormeta import PointGeneratorMeta
+from malcolm.core.stringmeta import StringMeta
+from malcolm.core.numbermeta import NumberMeta
+
+
+class ScanPointTickerController(Controller):
+
+    def __init__(self, block):
+        super(ScanPointTickerController, self).__init__(block)
+        self.stateMachine = None
+        self.value = Attribute(NumberMeta("value", "value", numpy.float64))
+
+    def create_attributes(self):
+        self.generator = Attribute(
+            PointGeneratorMeta("generator", "Scan Point Generator"))
+        yield self.generator
+        self.axis_name = Attribute(StringMeta("axis_name", "Name of the axis"))
+        yield self.axis_name
+        self.exposure = Attribute(
+            NumberMeta("exposure", "Exposure time", numpy.float64))
+        yield self.exposure
+
+    @takes(PointGeneratorMeta("generator", "Generator instance"), REQUIRED,
+           StringMeta("axis_name", "Specifier for axis"), REQUIRED,
+           NumberMeta("exposure", "Detector exposure time", numpy.float64), REQUIRED)
+    def configure(self, generator, axis_name, exposure):
+        """
+        Configure the controller
+
+        Args:
+            generator(PointGenerator): Generator to create points
+            axis_name(String): Specifier for axis
+            exposure(Double): Exposure time for detector
+        """
+
+        self.generator.set_value(generator)
+        self.axis_name.set_value(axis_name)
+        self.exposure.set_value(exposure)
+
+    @Method.wrap_method
+    def run(self):
+        """
+        Start the ticker process
+
+        Yields:
+            Point: Scan points from PointGenerator
+        """
+
+        for point in self.generator.value.iterator():
+            self.value.set_value(point)
```

Comment:
notify_subscribers() again...

Maybe we need a notify=False argument to all the set_* methods?